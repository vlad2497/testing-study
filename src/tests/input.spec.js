import React from "react";
import Input from './../components/ui/input/index';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() })

describe('should render input', () => {
  it('should render', () => {
    const component = shallow(<Input/>)
    expect(component).toMatchSnapshot()
  })
})
