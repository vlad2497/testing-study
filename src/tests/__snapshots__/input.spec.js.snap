// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render input should render 1`] = `
<div
  className="input-wrapper"
>
  <input
    type="text"
  />
</div>
`;
